var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -152,7 +154,7 @@ def configure_args(self):
                 oapp("--with-wannier90-bins={0}".format(spec["wannier90"].prefix.bin))
                 oapp("--enable-connectors")
                 oapp("--with-dft-flavor=atompaw+libxc+wannier90")
-            else:
+            elif "@:9.8" in spec:
```

Comment:
Version range checks should be checked with `spec.satisfies()`.  See docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints.

Please correct here and at least line 187.

Reply:
Should be corrected now